server/routes/api/choropleth.py

```diff
@@ -35,15 +35,51 @@
   url_prefix='/tools/choropleth'
 )
 
-@bp.route('/api')
-def choropleth_api():
+@bp.route('/values')
+def choropleth_values():
     """Returns data for geographic subregions for a certain statistical 
             variable.
 
-    API Params: #TODO(iancostello): Split API calls up or move to client.
-        statVar -> The statistical variable to download, as a string.
-        perCapita -> Whether to return the per-capita value, as a string of a 
-                boolean.
+    API Params:
+        geoId -> The currently viewed geography to render, as a string.
```

Comment:
Per https://google.github.io/styleguide/pyguide.html#383-functions-and-methods, please use colons, not arrows, here and elsewhere.

We use arrows when specifying types.

Reply:
Updated